From 19b5a44b60c40599bbc9dd434d598501187ff0f7 Mon Sep 17 00:00:00 2001
From: Simon Glass <sjg@chromium.org>
Date: Tue, 21 Mar 2017 12:30:01 -0600
Subject: [PATCH 6/6] pylibfdt: Look for Python 2.7

Chrome OS uses Python 2.7 and the ebuild does not install python.pc.
Update the Makefile to look for python-2.7 instead of python.

Signed-off-by: Simon Glass <sjg@chromium.org>
---

 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ed95384..bc6bd53 100644
--- a/Makefile
+++ b/Makefile
@@ -119,7 +119,7 @@ SCRIPTS = dtdiff
 # We need both Python and swig to build pylibfdt.
 .PHONY: maybe_pylibfdt
 maybe_pylibfdt: FORCE
-	if pkg-config --cflags python >/dev/null 2>&1; then \
+	if pkg-config --cflags python-2.7 >/dev/null 2>&1; then \
 		if which swig >/dev/null 2>&1; then \
 			can_build=yes; \
 		fi; \
-- 
2.12.1.500.gab5fba24ee-goog

